componentLibraries/defaultLibrary/Signal/Sources&Sinks/SignalLogger.hpp

```diff
@@ -193,6 +193,14 @@ namespace hopsan {
                 }
             }
 
+            //Close and re-open file in append mode
```

Comment:
This is very strange, I have not had time to test it my self, but is there a specific reason to do this here, after the init code. I could not see that anything was written to the file. So maybe it could be opened in append mode directly. Or well, ofcourse it needs to be cleared first. Dont know if there is a clear and append mode. Or it could be reopened directly after the first open call. Probably does not matter, just curious if that would change anything. Or move the first open call down here. 

open -> close -> open again ....

Reply:
It does write file header for column-wise CSV. The other formats need to be buffered and is written later on. 

I'm not sure if this is related to buffer size or similar, since it is different on Windows and Linux...